transform/models/staging/stg_gharchive.sql

```diff
@@ -0,0 +1,7 @@
+select 
+  left(type, len(type) - 5) as event_type, 
+  actor.login as user, 
+  repo.id as repo_id, 
+  repo.name as repo_name, 
+  created_at as event_date 
+from {{ source("src_gharchive", "src_gharchive") }} 
```

Comment:
If we modify the name of the source cf. comment above, we need to modify it here too.